source/unified-test-format/schema-1.3.json

```diff
@@ -194,14 +210,47 @@
       "required": ["client", "events"],
       "properties": {
         "client": { "type": "string" },
-        "events": {
-          "type": "array",
-          "items": { "$ref": "#/definitions/expectedEvent" }
+        "eventType": {
+          "type": "string",
+          "enum": ["command", "cmap"]
+        },
+        "events": { "type": "array" }
+      },
+      "oneOf": [
+        {
+          "required": ["eventType"],
+          "properties": {
+            "eventType": { "const": "command" },
+            "events": {
+              "type": "array",
+              "items": { "$ref": "#/definitions/expectedCommandEvent" }
+            }
+          }
+        },
+        {
+          "required": ["eventType"],
+          "properties": {
+            "eventType": { "const": "cmap" },
+            "events": {
+              "type": "array",
+              "items": { "$ref": "#/definitions/expectedCmapEvent" }
+            }
+          }
+        },
+        {
+          "additionalProperties": false,
+          "properties": {
+            "client": { "type": "string" },
+            "events": {
+              "type": "array",
+              "items": { "$ref": "#/definitions/expectedCommandEvent" }
+            }
+          }
         }
-      }
+      ]
     },
 
-    "expectedEvent": {
+    "expectedCommandEvent": {
```

Comment:
Please rename all of the `expectedEvent` files in `invalid` to `expectedCommandEvent`.